.pre-commit-config.yaml

```
@@ -21,7 +21,8 @@ repos:
     exclude: |
       (?x)(
       \.svcg$ |
-      ^boefjes/tests/examples/rdns-nxdomain.txt$
+      ^boefjes/tests/examples/rdns-nxdomain.txt$ |
+      ^boefjes/tests/examples/raw/.*
```

The `.*` doesn't do anything

```suggestion
      ^boefjes/tests/examples/raw/
```